api/src/main/resources/api.yaml

```diff
@@ -755,8 +755,66 @@ paths:
           description: If house member was removed from house
         '400':
           description: if params are invalid
-
-
+  /houses/{houseId}/rentals:
+    post:
+      tags:
+        - house-controller
+      description: Add the duration of stay to house history.
+      operationId: captureStay
+      parameters:
+        - in: path
+          name: houseId
+          schema:
+            type: string
+          required: true
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/HouseHistoryRequest'
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/HouseHistoryRequest'
+        required: true
+      responses:
+        '201':
+          description: If duration of stay was added to house history
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HouseHistoryResponse'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/HouseHistoryResponse'
+        '404':
+          description: If parameters are invalid
+    get:
+      tags:
+        - house-controller
+      operationId: getHouseHistory
```

Comment:
`getHouseHistory`?
Maybe it would be better to name it like `getRentalHistoryForHouse`?